Networking/Networking/Model/Site.swift

```diff
@@ -114,6 +118,14 @@ public struct Site: Decodable, Equatable, GeneratedFakeable, GeneratedCopiable {
         let planContainer = try siteContainer.nestedContainer(keyedBy: PlanInfo.self, forKey: .plan)
         let plan = try planContainer.decode(String.self, forKey: .slug)
 
+        let isAIAssitantFeatureActive: Bool = {
+            guard let featuresContainer = try? planContainer.nestedContainer(keyedBy: Features.self, forKey: .features),
+                  let activeFeatures = try? featuresContainer.decode([String].self, forKey: .active)else {
```

Comment:
Nit: spacing needed:

```suggestion
                  let activeFeatures = try? featuresContainer.decode([String].self, forKey: .active) else {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10694/commits/026e89cd457529e1deaa3081671cb58b17c2e715